### PR TITLE
Feature v0.9.4 mac wg quick

### PR DESCRIPTION
**Pull request by afeiszli**
Removes dependency for wg-quick from MacOS client, allowing movement on an official package.

### CODE REVIEW

File: netclient/wireguard/mac.go [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+		return err
+	}
+	time.Sleep(time.Second / 2)
+	err = setConfig(realIface, confPath)
+	if err != nil {
+		ncutils.PrintLog("error setting config for "+realIface, 1)
+		return err
+	}
+	var ips []string
+	ips = append(node.AllowedIPs, node.Address)
```
Comment (0xdcarns) [inline netclient/wireguard/mac.go:58]: could do:
var ips = append(node.AllowedIPs, []string{node.Address, node.Address6}...) if you wanna get to 1 line

File: netclient/wireguard/mac.go [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+	var confCmd = "grep -v -e Address -e MTU -e PostUp -e PostDown "
+	confRaw, _ := ncutils.RunCmd(confCmd+path, false)
+	return confRaw
+}
+
+// SetMacPeerRoutes - sets routes for interface from the peer list for all AllowedIps
+func SetMacPeerRoutes(iface string) error {
+	var err error
+	realIface := iface
+	/*
```
Comment (0xdcarns) [inline netclient/wireguard/mac.go:236]: remove unnecessary comment?

**Review by 0xdcarns — COMMENTED** [review]
Fix these few things and then review and I think it's okay.

File: netclient/ncutils/netclientutils.go [hunk 3]
```diff
@@ -318,6 +319,38 @@ func GetNetclientPathSpecific() string {
 	}
 }
 
+func GetNewIface(dir string) (string, error) {
+	files, _ := ioutil.ReadDir(dir)
```
[the remark this answers is not in the document]
Comment (0xdcarns) [reply netclient/ncutils/netclientutils.go:323]: there should be no ioutil in code base

[the remark this answers is not in the document]
Comment (afeiszli) [reply netclient/ncutils/netclientutils.go]: fixed

**Comment by afeiszli** [comment]
added updates based on feedback

**Comment by afeiszli** [comment]
fixed some more comments

**Pull request merged**